libs/core-ui/src/lib/Highchart/bubbleChart/getScatterPlot.ts

```diff
@@ -97,16 +108,26 @@ function getMarker(selectedPointsIndexes: number[], index: number): IMarker {
   return marker;
 }
 
+function getColor(selectionIndex: number, colorToUse?: string): string {
+  let color;
+  if (colorToUse) {
+    color = colorToUse;
+  } else if (selectionIndex === -1) {
```

Comment:
This is not a magic number. This is an index of an element and if array doesn't have item we are searching for it returns -1. I don't think it is needed to make this as a constant. Please let me know you opinion if otherwise.